src/Core/Components/DataGrid/FluentDataGridCell.razor.cs

```diff
@@ -24,12 +24,18 @@ public partial class FluentDataGridCell<TGridItem> : FluentComponentBase
     [Parameter]
     public DataGridCellType? CellType { get; set; } = DataGridCellType.Default;
 
+    /// <summary>
+    /// Gets or sets the column of the cell.
+    /// </summary>
+    [Parameter]
+    public ColumnBase<TGridItem>? Column { get; set; }
+
     /// <summary>
     /// Gets or sets the column index of the cell.
     /// This will be applied to the css grid-column-index value applied to the cell.
     /// </summary>
     [Parameter]
-    public int GridColumn { get; set; }
+    public int ColumnIndex { get; set; }
```

Comment:
You cannot change the name of an existing property. This modification is a breaking change.

Reply:
Done